src/api/challenge.ts

```diff
@@ -1,15 +1,19 @@
-import { _ludexChallengeApi, ApiConfig } from './utils';
+import { _ludexChallengeApi, ApiConfig } from "./utils";
 
 export class ChallengeAPIClient {
   ludexChallengeApi: <T>(config: ApiConfig) => Promise<T>;
   constructor(apiKey: string, baseUrl?: string) {
     this.ludexChallengeApi = _ludexChallengeApi(apiKey, "challenge", baseUrl);
   }
 
-  async _apiCreateChallenge(payoutId: number, limit: number = 2) {
+  async _apiCreateChallenge(
```

Comment:
the _ prefixed one is the "private" one, needs a change on create at the bottom.

Reply:
Gotcha